projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -833,6 +849,11 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
         return this._inputDirective || this._inputDirectiveUserTemplates.first || null;
     }
 
+    /** @hidden @internal */
+    public get labelDirective(): IgxLabelDirective {
```

Comment:
there is field with same name, `labelDirective` on line 671.

Reply:
resolve merge master